assets/js/components/notifications/CoreSiteBannerNotifications.js

```diff
@@ -91,12 +91,8 @@ const CoreSiteBannerNotifications = () => {
 				__( 'OK, Got it!', 'google-site-kit' )
 			}
 			isDismissible={ notification.dismissible }
-			onCTAClick={ async () => {
-				await acceptNotification( notification.id );
-			} }
-			onDismiss={ async () => {
-				await dismissNotification( notification.id );
-			} }
+			onCTAClick={ () => acceptNotification( notification.id ) }
+			onDismiss={ () => dismissNotification( notification.id ) }
```

Comment:
Please, use `useCallback` hooks for these functions.

Reply:
https://github.com/google/site-kit-wp/pull/4349/commits/5497cbabd0c66d25fa51081c7cabe40f2be3cf1b